plugins/mac-avcapture/av-capture.mm

```diff
@@ -1401,6 +1401,15 @@ inline static void av_capture_defaults(obs_data_t *settings,
 
 	obs_data_set_default_bool(settings, "enable_audio",
 				  enable_audio_and_high_preset);
+
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 130000
+	if (__builtin_available(macOS 13.0, *)) {
+		obs_data_set_default_string(
+			settings, "device",
+			AVCaptureDevice.systemPreferredCamera.uniqueID
+				.UTF8String);
+	}
+#endif
```

Comment:
The issue with not consulting preferredCamera is a user may have an available continuity camera, maybe the one they last used, but it's in their pocket. While it was their last choice, defaulting to it would likely not be the desired experience.

Automatically defaulting to 'last choice' without consideration for wireless cameras is likely to exacerbate that issue.

Reply:
As discussed off-thread, this change doesn't seem necessary as we do not select a device on new video input sources by default (waiting for users to make an actual choice) and this would also only affect _new_ video input sources-

Any existing video input source retains their device selection, so regardless of the then-current preferred camera, it would have no impact on existing sources. If a new OBS session is launched, the source will try to reconnect with the same device and depending on availability it will either show nothing (device not found) or whatever it is pointed at - and yes this might be the iPhone pointing at the insides of a trousers' pocket.

We _should_ make use of the system preferred and user preferred APIs in a second PR that enables "automatic camera selection" to mimic functionality of FaceTime, but that code path would then be used in the observer handling the device change.